test/unit/test_job.py

```diff
@@ -6,6 +6,7 @@
 from pathlib import Path
 
 import pytest
+from typing import Callable
```

Comment:
Not used? Or maybe you were going to add it to a function, but realized this was an older test?